src/main/java/org/broadinstitute/consent/http/service/InstitutionService.java

```diff
@@ -60,7 +65,13 @@ public void deleteInstitutionById(Integer id) {
   public Institution findInstitutionById(Integer id) {
     Institution institution = institutionDAO.findInstitutionById(id);
     isInstitutionNull(institution);
-    return institutionDAO.findInstitutionById(id);
+
+    List<SimplifiedUser> signingOfficials = userDAO.getSOsByInstitution(id).stream()
+            .map(SimplifiedUser::new)
+            .collect(Collectors.toList());
+    institution.setSigningOfficials(signingOfficials);
```

Comment:
An optimization here would be to combine the two DAO calls into one. That would be a somewhat larger refactor as it would touch the mappers and reducers. This works, however.